microsoft-edge/devtools-guide-chromium/device-mode/testing-other-browsers.md

```diff
@@ -23,56 +23,55 @@ ms.date: 07/16/2024
 ## Browser emulators
 
-Browser emulators are great for testing the responsiveness of a site.  But a browser emulator doesn't emulate differences in API, CSS support, and certain behaviors that manifest only on a mobile browser on an actual device.  Test your site on browsers running on real devices, to be certain everything behaves as expected.
+Browser emulators are great for testing the way your site looks and behaves at different screen sizes and resolutions, or when touch events are enabled, or when the user-agent string is set to another browser's user-agent string.
 
+You use browser emulators from the browser you're developing on. For example, in Microsoft Edge, you can use the **Device Emulation** tool in DevTools to get an approximation of how your site looks and behaves on different devices.  See [Emulate mobile devices (Device Emulation)](./index.md) for more information on how to use the **Device Emulation** tool in Microsoft Edge DevTools.
 
-<!-- ------------------------------ -->
-#### Firefox Responsive Design View
+Browser emulators don't emulate differences in web API or CSS support. For example, the **Device Emulation** tool in Microsoft Edge doesn't emulate how the different web APIs or CSS features that your site might be using are supported in Firefox or Safari. Test your site on browsers running on real devices, to be certain everything behaves as expected.
+
+Here are some browser emulators you can use to test your site on other browsers:
 
-Firefox has a [responsive design view](https://developer.mozilla.org/docs/Tools/Responsive_Design_View) that encourages you to stop thinking in terms of specific devices and instead explore how your design changes at common screen sizes, or on your own screen size by dragging the edges of the window.
+* [Firefox Responsive Design Mode](https://firefox-source-docs.mozilla.org/devtools-user/responsive_design_mode/index.html).
+* [Chrome Device Mode](https://developer.chrome.com/docs/devtools/device-mode).
+* [Safari Responsive Design Mode](https://developer.apple.com/documentation/safari-developer-tools/responsive-design-mode).
 
 
 <!-- ====================================================================== -->
 ## Device emulators and simulators
 
-Device simulators and emulators simulate not just the browser environment but the entire device.  Each simulator is useful to test things that require OS integration, such as form input with virtual keyboards.
+Device simulators and emulators simulate not just the browser environment but the entire device.  Each simulator is useful to test things that require specific integration with the operating system, such as form input with mobile device keyboards.
+
+You install device emulators and simulators on your development machine. You can therefore continue to test your website on the same machine where you're developing, while simulating the experience of using a different device.
 
```

Comment:
a local commit that didn't get pushed yet, probably from a PB comment: integrate/check:
![image](https://github.com/user-attachments/assets/feeeaa73-f517-4e18-8103-660e4d730019)

If Device Emulation in Microsoft Edge DevTools isn't sufficient for your testing needs, you can test by using real devices instead.  If you don't have access to the real devices that you want to test on, setting up a virtual machine or using a cloud-based testing service can help.  A big advantage of cloud-based testing services is that they can help you automate testing for your site across different platforms and form factors.

@captainbrosset